core/src/test/java/org/infinispan/remoting/AsynchronousInvocationTest.java

```diff
@@ -99,9 +96,6 @@ public void setUp() throws Throwable {
       commandsFactory = extractCommandsFactory(cache);
 
       //populate commands
-      blockingCacheRpcCommand = new StreamRequestCommand<>(cacheName, null, null, false,
-                                                           StreamRequestCommand.Type.TERMINAL, null, null, null, false,
-                                                           false, mock(TerminalOperation.class));
```

Comment:
Is there no other blocking `CacheRpcCommand` we could have used instead?

Reply:
I tried a few, but couldn't find one that still passed the assertion. Do you know of one we could add back in? To be honest I would hope we would eventually get to a point where no command would block, as we would handle it in the interceptors.

Reply:
`CompleteTransactionCommand`

Reply:
I created #7962 to use mocks for all command types, since the test is about `InboundInvocationHandler`.